src/managers/Queue.ts

```diff
@@ -261,7 +261,7 @@ export class Queue {
     async playlist(search: Playlist | string, options: PlaylistOptions & { data?: any } = DefaultPlaylistOptions): Promise<Playlist> {
         if(this.destroyed)
             throw new DMPError(DMPErrors.QUEUE_DESTROYED);
-        if(!this.connection?.connection)
+        if(!this.connection)
```

Comment:
IMO `if (!this.connection)` is clearer than `if (!this.connection?.connection)` because `this.connection` is the thing that can be `undefined` (`StreamConnection#connection` is never `undefined`).